node/baremetal/README.md

```diff
@@ -0,0 +1,418 @@
+```
+
+5. Create a JWT secret for Geth
+
+```bash
+cd /opt/ethereum/prysm
+sudo ./prysm.sh beacon-chain generate-auth-secret
+```
+
+6. Configure systemd to run Geth
+
+Add a new file called `/etc/systemd/system/geth.service`:
+
+```bash
+[Unit]
+Description=Geth Mainnet Ethereum Client
+Wants=prysm.service
+
+[Service]
+ExecStart=/usr/bin/geth --syncmode "snap" \
+        --http \
+        --http.api "eth,net,web3,personal" \
+        --cache=2048 \
+        --maxpeers=25 \
+        --datadir PATHTO/ethereum/geth/data \
+        --port 30303 \
+        --http.port 8545 \
+        --authrpc.jwtsecret /opt/ethereum/prysm/jwt.hex
+Restart=always
+RestartSec=60
+TimeoutSec=180
+KillMode=process
+
+[Install]
+WantedBy=multi-user.target
+```
+
+7. Configure systemd to run Prysm
+
+Add a new file called `/etc/systemd/system/prysm.service`:
+
+```bash
+[Unit]
+Description=Prysm Mainnet Consensus Client
+After=network.target
+After=geth.service
+Wants=geth.service
+Requires=network.target
+StartLimitIntervalSec=0
+
+[Service]
+ExecStart=/bin/bash /opt/ethereum/prysm/prysm.sh beacon-chain \
+        --execution-endpoint=http://localhost:8551 \
+        --jwt-secret=PATHTO/ethereum/prysm/jwt.hex \
+        --checkpoint-sync-url=https://sync-mainnet.beaconcha.in \
+        --genesis-beacon-api-url=https://sync-mainnet.beaconcha.in \
+        --datadir=PATHTO/ethereum/prysm/data
+WorkingDirectory=/opt/ethereum/prysm
+Restart=always
+RestartSec=60
+TimeoutSec=180
+KillMode=process
+Environment="USE_PRYSM_MODERN=true"
+
+[Install]
+WantedBy=multi-user.target
+```
+
+8. Enable the new services to run with the following commands:
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl enable geth.service
+sudo systemctl enable prysm.service
+sudo systemctl start geth.service
+```
+
+The first command refreshes systemd to recognize the new services, and the second
+command enables the services to run on startup. The third command starts the
+services. You only need to call start on geth.service as it is set with
+`Wants=prysm.service` and will trigger the startup of that service.
+
+Before moving forward you can confirm the two services have started up using the
+following command for geth:
+
+```bash
+sudo systemctl status geth
+```
+
+and the following command for prysm:
+
+```bash
+sudo systemctl status prysm
+```
+
+8. Wait for the L1 node to sync. This can take a day or so to fully catch up
+   with the latest blocks.
+
+## Set up Base Node
+
+Base nodes are run as a combination of two clients. One is [OP-Geth](https://github.com/ethereum-optimism/op-geth),
+which is a forked version of Geth that does the block creation for the L2. The
+other is the [OP-Node](https://github.com/ethereum-optimism/optimism/tree/develop/op-node),
+which handles the rollup processing from L1 sources as well as the L2 -> L1 data feed.
+
+In order to run all of these as root, you will need to add `go` to the root path:
+
+1. Edit the sudo config: `sudo visudo`
```

Comment:
This feels complicated. Any way to simplify?

Reply:
No, running this as root requires usage of Go for the installation to store it in the `/opt` directory. The way golang is installed this is unfortunately required.